docs/aea-vs-mvc.md

```diff
@@ -0,0 +1,16 @@
+The AEA borrows several concepts from popular web frameworks like Django and Ruby on Rails.
+
+Both aforementioned web frameworks use the popular MVC (model-view-controller) architecture.
+
+- Models: contain business logic and data representations
+- View: contain the html templates
+- Controller: deals with the request-response handling
+
+The AEA framework is based on asynchroneous messaging. Hence there is not a direct 1-1 relationship between MVC based architectures and the AEA framework. Nevertheless, there are some parallels which can help a developer familiar with MVC make progress in the AEA framework, in particular the development of Skills, quickly:
```

Comment:
- Asynchronous over asynchroneous.
- I think it needs a comma `,` after in particular.